packages/cardano-services/src/cli.ts

```diff
@@ -101,16 +105,40 @@ commonOptions(
     (interval) => Number.parseInt(interval, 10),
     DB_POLL_INTERVAL_DEFAULT
   )
+  .option(
+    '--service-discovery-backoff-factor <serviceDiscoveryBackoffFactor>',
+    ProgramOptionDescriptions.ServiceDiscoveryBackoffFactor,
+    (factor) => Number.parseFloat(factor),
+    SERVICE_DISCOVERY_BACKOFF_FACTOR_DEFAULT
+  )
+  .option(
+    '--service-discovery-timeout <serviceDiscoveryTimeout>',
+    ProgramOptionDescriptions.ServiceDiscoverBackoffTimeout,
+    (interval) => Number.parseInt(interval, 10),
+    SERVICE_DISCOVERY_BACKOFF_MAX_TIMEOUT_DEFAULT
+  )
   .option('--use-queue', ProgramOptionDescriptions.UseQueue, () => true, USE_QUEUE_DEFAULT)
+  .option('--rabbitmq-srv-service-name <rabbitmqSrvServiceName>', ProgramOptionDescriptions.RabbitMQSrvServiceName)
+  .option('--ogmios-srv-service-name <ogmiosSrvServiceName>', ProgramOptionDescriptions.OgmiosSrvServiceName)
+  .option('--postgres-srv-service-name <postgresSrvServiceName>', ProgramOptionDescriptions.PostgresSrvServiceName)
+  .option('--postgres-name <postgresName>', ProgramOptionDescriptions.DbConnection)
+  .option('--postgres-uid <postgresUser>', ProgramOptionDescriptions.DbConnection)
+  .option('--postgres-password <postgresPassword>', ProgramOptionDescriptions.DbConnection)
   .action(async (serviceNames: ServiceNames[], options: { apiUrl: URL } & HttpServerOptions) => {
     const { apiUrl, ...rest } = options;
-    const server = await loadHttpServer({ apiUrl: apiUrl || API_URL_DEFAULT, options: rest, serviceNames });
-    await server.initialize();
-    await server.start();
-    onDeath(async () => {
-      await server.shutdown();
+    try {
```

Comment:
Align it with `run.ts` - `loadHttpServer` is wrapped within a try-catch block

Reply:
Good catch :muscle: